packages/slate-editor/src/modules/nodes-hierarchy/fixers/convertToParagraph.ts

```diff
@@ -12,6 +17,10 @@ export function convertToParagraph(editor: Editor, [node, path]: NodeEntry) {
         children.push(node);
     }
 
+    if (children.length === 0) {
+        children.push({ text: '' });
+    }
```

Comment:
I don't think this will work. 

IIRC, `Transforms.setNodes()` only sets node attributes and completely ignores `children`.

Reply:
https://github.com/ianstormtaylor/slate/blob/fe13a8f9e750569342ee004951e34233ab6614bf/packages/slate/src/transforms/node.ts#L658-L674

Reply:
#267